gallery/setupTests.js

```diff
@@ -12,7 +12,12 @@ jest.retryTimes(3);
 
 const toMatchImageSnapshot = configureToMatchImageSnapshot({
   customDiffDir: path.join(__dirname, 'test-results'),
-  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier
+  // eslint-disable-next-line no-undef
+  customSnapshotsDir: process.env.RSC_GALLERY_THEME === 'DARK'
+  && path.join(__dirname, 'visualtests', '__image_snapshots_dark__'),
+  customSnapshotIdentifier: ({ defaultIdentifier }) =>
+    // eslint-disable-next-line no-undef
```

Comment:
These eslint disables don't see necessary on my system

Reply:
> These eslint disables don't see necessary on my system

Actually I don't think this file is getting linted at all.  The various lint scripts in `package.json` are all constrained to files in `src` and `visualtests`, and this file is in neither of those directories.  It should get included in the test linting.

Reply:
> > These eslint disables don't see necessary on my system
> 
> Actually I don't think this file is getting linted at all. The various lint scripts in `package.json` are all constrained to files in `src` and `visualtests`, and this file is in neither of those directories. It should get included in the test linting.

This had been a lintr error prior to merging RSC-1118 into FEAUTRE_RSC-928, but now that all visual test files are overridden in the gallery's eslintrc file, it's no longer needed

Reply:
<img width="624" alt="Screen Shot 2022-10-25 at 2 16 17 PM" src="https://user-images.githubusercontent.com/89862727/197850668-85c958ae-2c4b-4454-a0b2-60427b548cee.png">
Unfortunately, I am still getting it on my end, and checked to see that my branch was currently up to date. You suggested in https://github.com/sonatype/sonatype-react-shared-components/pull/808#discussion_r1003251307 to override the lintr errors in setupTest.js. Would this override cancel out the benefits of adding `setupTest.js` to the lint scripts in package.json ?